### PR TITLE
feat: change price with product configuration

### PR DESCRIPTION
**Pull request by Francois-Gomis**

### CODE REVIEW

File: view/frontend/web/js/view/catalog/badge.js [hunk 1]
```diff
@@ -80,6 +92,30 @@ define([
...
     }
 
+    /**
+     *
+     * @param productId Id of product for Id concatenation
+     * @param subClass Sub class to select
+     * @returns {*}
+     */
+    function getHtmlPriceContainer(productId,subClass = false){
+        var classToSelect = ''
```
Comment (syjust-alma) [inline view/frontend/web/js/view/catalog/badge.js:102]: I know `;` are not required ... but can you add it anyway please

File: view/frontend/web/js/view/catalog/badge.js [hunk 2]
```diff
@@ -53,24 +57,32 @@ define([
         }
     }
 
-    function moveToCustomPosition(customDisplay,containerId){
+    /**
+     *
+     * @param customDisplay Widget config define in view/frontend/templates/catalog/product/view.phtml
+     * @param baseContainerId Base container who contain the widget
+     */
+    function moveToCustomPosition(customDisplay,baseContainerId){
```
Comment (syjust-alma) [inline view/frontend/web/js/view/catalog/badge.js:65]: can you add ` ` (space) after `,` (comma) please  😉

File: view/frontend/web/js/view/catalog/badge.js [hunk 3]
```diff
@@ -80,6 +92,30 @@ define([
         return price ;
     }
 
+    /**
+     *
+     * @param productId Id of product for Id concatenation
+     * @param subClass Sub class to select
+     * @returns {*}
+     */
+    function getHtmlPriceContainer(productId,subClass = false){
```
Comment (syjust-alma) [inline view/frontend/web/js/view/catalog/badge.js:101]: can you add ` ` (space) after `,` (comma) please  😉

**Review by syjust-alma — COMMENTED** [review]
Just minor code style comments.
Everything seem's good 😉 

GG

File: view/frontend/web/js/view/catalog/badge.js (hunk 1 above)
Comment (Francois-Gomis) [reply view/frontend/web/js/view/catalog/badge.js:102]: Done

File: view/frontend/web/js/view/catalog/badge.js (hunk 2 above)
Comment (Francois-Gomis) [reply view/frontend/web/js/view/catalog/badge.js:65]: Done

File: view/frontend/web/js/view/catalog/badge.js (hunk 3 above)
Comment (Francois-Gomis) [reply view/frontend/web/js/view/catalog/badge.js:101]: Done

**Pull request merged**